### PR TITLE
Initialize Feedstock

### PR DESCRIPTION
**Pull request by SC426**
Jira ticket: https://anaconda.atlassian.net/browse/PKG-842
Upstream repo: https://github.com/frostming/findpython
Changelog: https://github.com/frostming/findpython/releases

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,9 +37,13 @@ test:
 
 about:
   home: https://pypi.org/project/findpython/
+  description: A utility to find python versions on your system
   summary: A utility to find python versions on your system
   license: MIT
   license_file: LICENSE
+  license_family: MIT
+  dev_url: https://github.com/frostming/findpython
+  doc_url: https://github.com/frostming/findpython
```
Comment (marcoesters) [inline recipe/meta.yaml]: This is a better doc url: https://github.com/frostming/findpython/blob/main/README.md

File: recipe/meta.yaml [hunk 2]
```diff
@@ -35,9 +37,13 @@ test:
 
 about:
   home: https://pypi.org/project/findpython/
+  description: A utility to find python versions on your system
```
Comment (marcoesters) [inline recipe/meta.yaml]: Maybe just copy the upstream description (without the "2@' symbol)

**Review by marcoesters — COMMENTED** [review]
Please remove the abs.yaml file

File: recipe/meta.yaml (hunk 1 above)
Comment (SC426) [reply recipe/meta.yaml]: This has been updated

File: recipe/meta.yaml (hunk 2 above)
Comment (SC426) [reply recipe/meta.yaml]: This has been updated

**Review by marcoesters — APPROVED** [review]
LGTM

**Review by ELundby45 — APPROVED** [review]
LGTM

**Pull request merged**